test/k8s-e2e/e2e-test-execute

```diff
@@ -430,15 +428,15 @@ function pushResutlsToGCS() {
 
 	# Upload results
 	# - Use GCE instead of GCP, since Google uses it on https://k8s-testgrid.appspot.com/conformance-gce
-	# - v${K8S_RELEASE%*\.[0-9]*} cuts off patch release and returns v1.10, v1.11
 	if [ "$CLOUDPROVIDER" = "gcp" ]; then
 		CLOUDPROVIDER_PUBLIC="gce"
 	elif [[ "$CLOUDPROVIDER" = os-* ]]; then
 		CLOUDPROVIDER_PUBLIC="os"
 	else
 		CLOUDPROVIDER_PUBLIC=${CLOUDPROVIDER}
 	fi
-		upload_res="$(python2 ./test-infra/testgrid/conformance/upload_e2e.py --junit="$log_dir/junit_01.xml" --log="$e2e_log" --bucket=gs://k8s-conformance-gardener/ci-gardener-e2e-conformance-"${CLOUDPROVIDER_PUBLIC}"-v"${K8S_RELEASE%*\.[0-9]*}" --metadata='{"commit-id":"'$commit_id'","shoot-k8s-release":"'$K8S_RELEASE'"}' --key-file=$google_bucket_secrets_file)"
+
+	upload_res="$(python2 ./test-infra/testgrid/conformance/upload_e2e.py --junit="$log_dir/junit_01.xml" --log="$e2e_log" --bucket=gs://k8s-conformance-gardener/ci-gardener-e2e-conformance-"${CLOUDPROVIDER_PUBLIC}"-v"${K8S_VERSION%*\.[0-9]*}" --metadata='{"commit-id":"'$GARDENER_COMMIT_ID'","shoot-k8s-release":"'$K8S_VERSION'"}' --key-file=$google_bucket_secrets_file)"
```

Comment:
Do you need a commit ID or is a version ok?

Reply:
doesn't matter which. This is just additional optional information to comprehend the results.